loadtest/stress.py

```diff
@@ -176,9 +176,25 @@ def test_storage_session(self):
             response = self.session.get(url, params=params, **reqkwds)
             self.assertTrue(response.status_code in (200, 404))
 
-        # PUT requests with several WBOs batched together
+        # POST requests with several WBOs batched together
         num_requests = self._pick_weighted_count(post_count_distribution)
-        cols = random.sample(collections, num_requests)
+        # Shallower distribution(for now) of "transactional" style batch
+        # uploads.  Looking for roughly 10% of batch POSTs to be
+        # "transactional".  Increase the frequency of "transactional" requests
+        # by changing the modulo 10 to something like modulo 5(~20%) or
+        # modulo 3(~30%).
+        # transact = int(random.paretovariate(1) * 100) % 10 == 0
+        #
+        # Let's do roughly 50% for fun.
+        transact = random.randint(0, 1)
```

Comment:
does this change mean the comment above it about the "modulo blah" is now obsolete?